recipes/default.rb

```diff
@@ -28,6 +28,11 @@
 
 directory '/etc/exabgp'
 
+template '/etc/exabgp/exabgp.sh' do
+  mode 0o0755
+  notifies :restart, 'service[exabgp]' if node['exabgp']['systemd']['enabled']
```

Comment:
What if not enabled? :) As I mentioned before, it's not our use case, but let's keep it the same way as all other resources in this cookbook

```
  notifies :run, 'execute[restart-exabgp]' unless node['exabgp']['systemd']['enabled']
  notifies :restart, 'service[exabgp]' if node['exabgp']['systemd']['enabled']
```